src/components/icons/Chip.tsx

```diff
@@ -0,0 +1,35 @@
+import { IIconProps } from './type';
+
+const Chip = ({ width = '35', height = '35', color = 'white' }: IIconProps) => {
```

Comment:
color prop is noteven used. please add a rule in eslint config so that it highlights these basic issues